pkg/healthcheck/healthcheck.go

```diff
@@ -1415,6 +1419,19 @@ func (hc *HealthChecker) allCategories() []*Category {
 			},
 			false,
 		),
+		NewCategory(
+			LinkerdOpaquePortsDefinitionChecks,
+			[]Checker{
+				{
+					description: "the opaque-ports labels have been properly defined",
```

Comment:
again, very nitpicky:
```suggestion
					description: "the opaque-ports annotations have been properly defined",
```